This should go to logs
Object (fromList [("error",Number 1.0)])
{"errorType":"ErrorCall","errorMessage":"Magic error\nCallStack (from HasCallStack):\n  error, called at Main.hs:25:30 in main:Main"}
<Serverless error omitted>
